python/core/utils/scratch_tests.py

```
@@ -1,66 +1,72 @@
     input_names = [i.name for i in model.graph.input]
 
     extract_model(
         input_path=model_path,
         output_path=save_path,
         input_names=input_names,
-        output_names=output_names
+        output_names=output_names,
     )
 
-    print(f"Pruned model saved to {save_path}")
+    print(f"Pruned model saved to {save_path}")  # noqa: T201
```

Ruff (0.14.7)

Unused `noqa` directive (non-enabled: `T201`)

Remove unused `noqa` directive

(RUF100)